### PR TITLE
Add Apache Guacamole Icon

### LINKED ISSUE #7649: Request: Apache Guacamole
### Brand Name

Simple Icons Apache Guacamole

### Website

https://guacamole.apache.org

### Popularity Metric

Commonly used in homelab

### Official Resources for Icon and Color

https://guacamole.apache.org

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by zrfisaac**
apacheguacamole

**Issue:** Close #7649

**Similarweb rank:** [--](https://www.similarweb.com/website/guacamole.apache.org/#overview)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
Apache Guacamole is a clientless remote desktop gateway. It supports standard protocols like VNC, RDP, and SSH.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4565,6 +4565,11 @@
             "source": "https://commons.wikimedia.org/wiki/File:GTK_logo.svg",
             "guidelines": "https://foundation.gnome.org/logo-and-trademarks/"
         },
+        {
+            "title": "Guacamole",
+            "hex": "304730",
+            "source": "https://github.com/apache/guacamole-website/blob/7b832798bb43f9047d0725b10604e09fd3245cbd/images/guac-logo.png"
```
Comment (adamrusted) [inline _data/simple-icons.json:4571]: Official SVG source is available at [this link](https://www.apache.org/logos/#guacamole) which can be used for this.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -4565,6 +4565,11 @@
             "source": "https://commons.wikimedia.org/wiki/File:GTK_logo.svg",
             "guidelines": "https://foundation.gnome.org/logo-and-trademarks/"
         },
+        {
+            "title": "Guacamole",
```
Comment (adamrusted) [inline _data/simple-icons.json:4569]:
```suggestion
            "title": "Apache Guacamole",
```

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Review by PratyushJayachandran — APPROVED** [review]
Thank you for the contribution @zrfisaac and the correction @adamrusted 
However I see [some slight differences!](https://github.com/simple-icons/simple-icons/assets/35000807/83718c14-9705-4589-bd96-930dabad9157)
However, we can merge it if you are okay with it @adamrusted

**Comment by PratyushJayachandran** [comment]
> Thank you for the contribution @zrfisaac and the correction @adamrusted However I see [some slight differences!](https://github.com/simple-icons/simple-icons/assets/35000807/83718c14-9705-4589-bd96-930dabad9157) However, we can merge it if you are okay with it @adamrusted

Merging this one as the differences are very minor!

**Pull request merged**